pkg/sdk/poc/example/sdk_definitions.go

```diff
@@ -30,16 +30,19 @@ type (
 )
 
 func randomAccountObjectIdentifier(t *testing.T) AccountObjectIdentifier {
+	t.Helper()
```

Comment:
this helpers are not necessary here, they are not used (check file description on the top)